package.json

```diff
@@ -9,6 +9,7 @@
     "lib"
   ],
   "scripts": {
+    "precommit": "npm run test",
```

Comment:
I'm not sure if I like `precommit` linting since I already have the linter on my text editor 😅 
What I like is the same thing we have on Hyper: `prepush: npm run test` 💯